addon/components/frost-object-browser.js

```diff
@@ -45,11 +48,21 @@ export default Component.extend({
   // == Actions ===============================================================
   actions: {
     collapseFilter () {
-      this.onFilterVisibleChange(false)
+      this.set('isFilterHiddenOnLoad', true)
```

Comment:
The grammar of this functionality the we need to expose to the consumer (it is something happening on load only) vs the grammar we need internally to manage state (isHidden) require two different words/descriptions though are dealing with the same variable.

To address this create a `isHidden` property that is internal to this component.  In the `init()` set the value of `isFilterHiddenOnLoad` on `isHidden` and then use that in places such as this through out the code.  This will keep the external and internal grammar separate, as well as it sets us up for future success is we need to do something that requires these worlds to be separated.